### PR TITLE
Allow passing in a rowIndex instead of just previous

### PR DESCRIPTION
**Pull request by marcodejongh**

### CODE REVIEW

File: src/table.js [hunk 1]
```diff
@@ -277,9 +277,9 @@ export const addColumnAt = columnIndex => tr => {
   return tr;
 };
 
-// :: (rowIndex: number, clonePreviousRow?: boolean) → (tr: Transaction) → Transaction
-// Returns a new transaction that adds a new row at index `rowIndex`. Optionally clone the previous row.
-//
+// :: (rowIndex: number, cloneIndex?: boolean | number) → (tr: Transaction) → Transaction
```
Comment (eshvedai) [inline src/table.js]: I think you should use union here, this line fails the build

Comment (eshvedai) [reply src/table.js]: try cloneIndex: ?union< boolean, number>

File: src/table.js [hunk 2]
```diff
@@ -291,11 +291,15 @@ export const addColumnAt = columnIndex => tr => {
 //   addRowAt(i, true)(state.tr)
 // );
 // ```
-export const addRowAt = (rowIndex, clonePreviousRow) => tr => {
+export const addRowAt = (rowIndex, cloneIndex) => tr => {
```
Comment (eshvedai) [inline src/table.js]: now you need to fix some existing tests and add more tests for the new behaviour 😏

File: src/table.js (hunk 1 above)
Comment (marcodejongh) [reply src/table.js]: Done

**Comment by eshvedai** [comment]
closing as it was abandoned

**Pull request closed without merging**